frontend/src/components/ReportSelect/index.tsx

```diff
@@ -139,7 +144,7 @@ export const ReportSelect = () => {
               }
             } else {
               // If the request type is route, we will add that value to the route params.
-              routeParam = value;
+              routeParam = value.value;
```

Comment:
are we sure about this, i feel like something might break here. @mhaswell-bcgov or @ASpiteri-BCGov  could one of you verify this.